File: 준영/2026_05_14/jadenCase.java
class Solution {
    public String solution(String s) {
        String answer = "";
        char[] arr = s.toLowerCase().toCharArray();
        for (int i = 0; i < arr.length; i++) {
            if (i == 0 && arr[i] >= 'a' && arr[i] <= 'z') {
                arr[i] = (char) (arr[i] - 32);
            } else if (i != 0 && arr[i - 1] == ' ' && arr[i] >= 'a' && arr[i] <= 'z') {
                arr[i] = (char) (arr[i] - 32);
            }
        }
        answer = new String(arr);
        return answer;
    }
}

File: 인범/2026_05_14/JadenCase.py
def solution(s):
    answer = []
    flag = True
    for item in s:
        if item == " ":
            flag=True
            answer.append(item)
            continue
        if flag:
            item = item.upper()
            answer.append(item)
            flag=False
        else :
            item = item.lower()
            answer.append(item)
            
            
    return ''.join(answer)